waves-scripts

```diff
@@ -21,6 +21,9 @@ start_nodes () {
 
     # start elementsd, esplora (et al)
     nigiri start --liquid
+
+    # wait for everything to be started up
+    sleep 2
```

Comment:
:question: Did the minting fail randomly?

Reply:
Yes, got an error -28 with wallet loading :/